spec/models/mixins/purging_mixin_spec.rb

```diff
@@ -1,84 +1,81 @@
 require "spec_helper"
 
 describe PurgingMixin do
-  describe ".purge" do
-    let(:example_class) { PolicyEvent }
-    let(:example_associated_class) { PolicyEventContent }
-    let(:purge_date) { 2.weeks.ago }
+  let(:example_class) { PolicyEvent }
+  let(:example_associated_class) { PolicyEventContent }
+  let(:purge_date) { 2.weeks.ago }
+
+  describe ".purge_date" do
+    it "purge_date should not raise exception" do
+      allow(example_class).to receive(:purge_config).with(:keep_policy_events).and_return(120)
+      expect(example_class.purge_date).to be_within(1.second).of(120.seconds.ago.utc)
+    end
+  end
 
-    before do
-      events = (-2..2).collect do |date_modifier|
+  describe ".purge" do
+    let(:events) do
+      (-2..2).collect do |date_modifier|
         FactoryGirl.create(:policy_event, :timestamp => purge_date + date_modifier.days)
       end
-      @unpurged_ids = events[-2, 2].collect(&:id)
     end
+    let(:all_ids) { events.collect(&:id) }
+    let(:unpurged_ids) { all_ids[-2, 2] }
```

Comment:
Can this `let` just get used below where you accessed `all_ids[-2, 2]` directly?